### DISCUSSION TITLE
Masked TextBox

### DISCUSSION THREAD

**Discussion by Hassan198766** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#3720] [Q&A]
hello
I am a new user in the subject of programming with C# WPF  and I recently became acquainted with the Material Design component - in the program demo app, I searched for  Masked Text Box and cant find anything please some body help me. tanks

**Comment by corvinsz** [comment]
@Hassan198766 as of right now the MDIX library doesn't have a masked TextBox.
To my knowledge the maintainers are saying that this library a theming library, and not a control library, so I am not sure if such thing will ever get added, since Googles Material Design specs don't say anything about a masked TextBox.

**Reply by Hassan198766** [reply]
thanks for your response - don't need to define control- I mean the MDIX can add some method for masked textbox or define a class for it-actually I found a class for it, and I call it in the textbox fields of Material Design and it work perfectly now
